ee/models/license.py

```diff
@@ -58,6 +58,7 @@ class License(models.Model):
         "organizations_projects",
         "google_login",
         "dashboard_collaboration",
+        "ingestion_taxonomy",
```

Comment:
Btw, you'll need to create a PR on posthog-cloud to add this new feature to the cloud paid plans, but definitely **not a blocker here.**

Reply:
I'm looking around that repo `posthog-cloud` right? Where do I add that specifically? I can't seem to find mentions to other enterprise features on it